sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala

```diff
@@ -189,7 +189,7 @@ class DiscoveryImpl(
     val detail = if (in.detail.isEmpty) Nil else List(in.detail)
     val seeDocs = DocLinks(sdkName).forErrorCode(in.code).map(link => s"See documentation: $link").toList
     val messages = message :: detail ::: seeDocs ::: sourceMsgs
-    val logMessage = messages.mkString("\n\n")
+    val logMessage = messages.mkString("\t|\t")
```

Comment:
is the pipe `|` intentional?

Reply:
Yes, I thought I should put a delimiter just in case.

Reply:
It makes sense where tabs aren't rendered.